tests/test_operator.py

```diff
@@ -103,6 +103,21 @@ def test_multiple_steppers(self, expr, exp_uindices, exp_mods):
         assert(i.indices[i.function._time_position].modulo == exp_mods[i.function.name]
                for i in flatten(retrieve_indexed(i) for i in exprs))
 
+    @skipif('device')
```

Comment:
so what's the issue with these few skipped ones?

Reply:
oh, nothing relevant. Just that we are `asserting` against a certain IET structure, which clearly is different if we generate omp-offloading code